background.html

```diff
@@ -6,10 +6,14 @@
 
     <!-- include background scripts -->
     <script type="text/javascript" src="privly-applications/shared/javascripts/local_storage.js"></script>
+    <script type="text/javascript" src="privly-applications/shared/javascripts/storage.js"></script>
```

Comment:
I believe you can remove both the stylesheets in the background.html. They are causing an error in the background console.